CHANGELOG.md

```diff
@@ -0,0 +1,118 @@
+# Change Log
+All notable changes to this project are documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/).
+
+## [Unreleased](https://github.com/burzum/cakephp-file-storage/compare/2.0.0-rc1...2.0)
```

Comment:
The main disadvantage of doing it like this, is that you have toupdate it every time you release a new version.  Alternatively, we could just link to the main branch.

On the other hand it's most useful this way.